UI/properties-view.cpp

```diff
@@ -1399,6 +1400,27 @@ void OBSPropertiesView::AddGroup(obs_property_t *prop, QFormLayout *layout)
 	connect(groupBox, SIGNAL(toggled(bool)), info, SLOT(ControlChanged()));
 }
 
+QWidget *OBSPropertiesView::AddInfo(obs_property_t *prop)
+{
+	const char *desc = obs_property_description(prop);
+
+	QLabel *label = new QLabel(QT_UTF8(desc));
+
+	enum obs_info_type type = obs_property_info_type(prop);
+
+	switch (type) {
+	case OBS_INFO_NORMAL:
+		break;
+	case OBS_INFO_WARNING:
+		label->setObjectName("warningLabel");
```

Comment:
why not using `setThemeID(label,"warning")` this also you to add an option "good"

Reply:
This is what I found when I searched how Error and Warning were made in the settings of OBS.